From fde0704f46152b1d65ca73f3a221735123a2f75c Mon Sep 17 00:00:00 2001
From: Kevin Hao <kexin.hao@windriver.com>
Date: Tue, 15 Oct 2013 10:52:50 +0800
Subject: [PATCH 1/5] ocf: remove the unneeded invocation of daemonize()

In the current kernel, all kthread are created by the kthreadd thread
and the invocation of function daemonize() for the kthread function is
not needed anymore. Actually function daemonize() has already been killed
by commit c4144670 (kill daemonize()) in the latest kernel.

Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 crypto/ocf/random.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/crypto/ocf/random.c b/crypto/ocf/random.c
index 4bb773f..02eee6a 100644
--- a/crypto/ocf/random.c
+++ b/crypto/ocf/random.c
@@ -199,7 +199,6 @@ random_proc(void *arg)
 	spin_unlock_irq(&current->sigmask_lock);
 	sprintf(current->comm, "ocf-random");
 #else
-	daemonize("ocf-random");
 	allow_signal(SIGKILL);
 #endif
 
-- 
1.8.4.93.g57e4c17

